scripts/deploy.ts

```diff
@@ -110,6 +178,90 @@ async function main() {
   )
 }
 
+const legacyContractAddress = '0x9e7A15E77e5E4f536b8215aaF778e786005D0f8d'
```

Comment:
I think it should be defined with a `.env`, so you more likely don't forget to update it